src/main/java/hudson/plugins/sonar/SonarInstallation.java

```diff
@@ -256,6 +262,14 @@ public TriggersConfig getTriggers() {
     return triggers;
   }
 
+  public boolean useTokenProperty() {
```

Comment:
this should follow java bean conventions so it works fully with the https://plugins.jenkins.io/configuration-as-code plugin:

```suggestion
  public boolean isUseTokenProperty() {
```